src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const strCompare = (a, b) => {
```

Comment:
Не варто скорочувати і економити букви, плюс функції варто з дієслова розпочинати
```suggestion
  const compareStrings = (a, b) => {
```